tests/data/overflow.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0"?>
+<!-- just a sanity check: body collision rewritten to xml -->
+<configuration xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+               xsi:noNamespaceSchemaLocation="../../input/simulation_schema.xsd">
+    <fps>22</fps>
+    <video_length>25</video_length>
+    <delta_t>0.014</delta_t>
+    <end_time>10.0</end_time>
+
+    <cuboid>
+        <position>
+            <x>0.0</x>
+            <y>0.0</y>
+            <z>0.0</z>
+        </position>
+        <grid_dim>
+            <dimx>1</dimx>
+            <dimy>2</dimy>
+            <dimz>3</dimz>
+        </grid_dim>
+        <grid_spacing>1.1225</grid_spacing>
+        <temperature>0.1</temperature>          <!-- aka mean brownian motion -->
+        <mass>1</mass>
+        <velocity>
+            <x>0.0</x>
+            <y>0.0</y>
+            <z>0.0</z>
+        </velocity>
+        <type>0</type>
+    </cuboid>
+
+    <cuboid>
+        <position>
+            <x>15.0</x>
+            <y>15.0</y>
+            <z>0.0</z>
+        </position>
+        <grid_dim>
+            <dimx>800000000000000000</dimx>
```

Comment:
i dont know if overflow tests are that usefull. This basically just tests that integers cant exceed an arbitrary limit we chose oursevles. Furthermore every simulation with those parameters would take millions of years to finish, so its a bit dreamt up ^^

Maybe we should just stick to tests which verify, that all the inputted data from the XML-file is actually parsed correctly and arrives in the `SimulationParams` class

Reply:
OK fair enough hahaha